orca-queue/src/test/kotlin/com/netflix/spinnaker/orca/q/handler/CompleteStageHandlerTest.kt

```diff
@@ -829,7 +829,7 @@ object CompleteStageHandlerTest : SubjectSpek<CompleteStageHandler>({
   }
 
   describe("completing synthetic stages") {
-    listOf(SUCCEEDED, FAILED_CONTINUE).forEach { taskStatus ->
+    SUCCEEDED.let { taskStatus ->
```

Comment:
Do we still need the `.let {}` bit here and below since they are each just operating on the one status?